### PR TITLE
Create CONTRIBUTING.md with basic instructions for contributions

### PR DESCRIPTION
**Pull request by ivanhrabcak**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+Thank you for investing your time in contributing to my project! I really value your work and help.
+
+# Code - general rules
+I try to write code as cleanly as possible, I think readability is the most important thing when wrtiting code.
```
Comment (DasMatus) [inline CONTRIBUTING.md:4]: Can it be in plural, since this is a community effort?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+ 
+- Functionality that is provided to the user is in the form of a trait (a file in the `traits` module) that is implemented for the `Edupage` struct
+- Use the `Edupage::request` method if possible when requesting data from edupage instead of using the client directly
+- Try not to use `Map<String, Value>` - create types for json that is returned from edupage
+- Use the `EdupageError` error type for all errors
+- Do not use `.unwrap()` or code that can crash unless it is certain that it will not crash (check with `is_ok`/`is_some`/`if let`)
+- ... and others, you can read the code for more information, or read the next section that describes the structure of this project, which also includes additional instructions.
```
Comment (DasMatus) [inline CONTRIBUTING.md:11]: This should not be here.

**Review by DasMatus — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (ivanhrabcak) [reply CONTRIBUTING.md:11]: What do you suggest, where should this information be?

Comment (DasMatus) [reply CONTRIBUTING.md:11]: As I said before, it should be shortened to "...and others", since it's obvious where to look for more information.

**Pull request merged**